gulpfile.js

```diff
@@ -9,7 +9,7 @@ var uglify = require('gulp-uglify');
 var sourcemaps = require('gulp-sourcemaps');
 var ngAnnotate = require('gulp-ng-annotate');
 var gulpJscs = require('gulp-jscs');
-var karmaServer = require('karma').server;
+var Server = require('karma').Server;
```

Comment:
Fixed :) Couldn't just lowercase it since it was a constructor. That is how the karma docs show how to use it, but I think that I updated this to something better.